pkgs/tools/nix/nix-output-monitor/generated-package.nix

```diff
@@ -9,10 +9,10 @@
 }:
 mkDerivation {
   pname = "nix-output-monitor";
-  version = "2.1.2";
+  version = "2.1.3";
   src = fetchzip {
-    url = "https://code.maralorn.de/maralorn/nix-output-monitor/archive/v2.1.2.tar.gz";
```

Comment:
Is there a reason we are not fetching from the [GitHub repo](https://github.com/maralorn/nix-output-monitor/tree/v2.1.3) here?

The GitHub repo has proper tags present, but NOT GitHub releases. Though update scripts like `nix-update-script` should still work.